packages/vendure-plugin-google-cloud-tasks/src/cloud-tasks-job-queue.strategy.ts

```diff
@@ -53,7 +161,7 @@ export class CloudTasksJobQueueStrategy implements JobQueueStrategy {
           `Added job with retries=${cloudTaskMessage.maxRetries} to queue ${queueName}: ${cloudTaskMessage.id} for ${task.httpRequest.url}`,
           CloudTasksPlugin.loggerCtx
         );
-        return new Job({
+        const successfullJob = new Job({
```

Comment:
(I will pick up these changes, because I also have to test it locally anyway. But for future reference:)

I think this isn't a successfull job yet, it only pushed a job successfully to the Queue. We actually want to log it when the handling of the job succeeded, which happens in the controller.